frontend/src/concepts/pipelines/apiHooks/mlmd/usePipelineRunMlmdContext.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Context } from '~/third_party/mlmd';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import useFetchState, {
+  FetchState,
+  FetchStateCallbackPromise,
+  NotReadyError,
+} from '~/utilities/useFetchState';
+import { GetContextByTypeAndNameRequest } from '~/third_party/mlmd/generated/ml_metadata/proto/metadata_store_service_pb';
+import { FAST_POLL_INTERVAL } from '~/utilities/const';
+
+const KFP_V2_RUN_CONTEXT_TYPE = 'system.PipelineRun';
+
+const useMlmdContext = (
+  name?: string,
+  type?: string,
```

Comment:
Is there a known set of MLMD context types we can make a typescript type for here?

Reply:
Not as I know of after checking the kubeflow code https://github.com/opendatahub-io/data-science-pipelines/blob/680e4452ceaa209f80c6fe8d689a02aaded786f7/frontend/src/mlmd/MlmdUtils.ts#L55, we can type them more when we get to pass more and more types into it later, but it should be good for now.

Reply:
I don't see why we would offer a type parameter in a file scoped to pipeline runs, when it seems the only type one would use here is the one you just referenced from KF's codebase.

Reply:
Looks like the file might have moved, but either way, it makes more sense to me to add the 1 type we're using now to an enum that can be expanded upon later.